jsapp/js/components/newFeatureDialog.module.scss

```diff
@@ -0,0 +1,49 @@
+@use '~kobo-common/src/styles/colors';
+
+.root {
+	position: relative;
+  z-index: 1100; // 1 index lower than a modal
+}
+
+.wrapper {
+	position: relative;
+}
+
+.dialog {
+  width: 50%;
+  position: absolute;
+  border-radius: 4px;
+  padding: 12px;
+  color: colors.$kobo-white;
+  background: colors.$kobo-dark-blue;
+}
+
+.dialog::before {
+  content: " ";
+  left: 5%;
+  border: solid transparent;
+  height: 0;
+  width: 0;
+  position: absolute;
+  pointer-events: none;
+  border-width: 6px;
+  margin-left: calc(6px * -1);
+  bottom: 100%;
+  border-bottom-color: colors.$kobo-dark-blue;
+}
+
+.header {
+  color: colors.$kobo-white !important;
+  font-weight: 600 !important;
```

Comment:
Do these two lines need the `!important`? The styling appears to be the same without it (although that could be down to differences in browser/platform.)

Generally, we want to avoid using `!important` - if you need it, your CSS selector could probably stand to be more specific (`div.header` etc.)